gnc/sub8_perception/nodes/follow_orange_pipes.py

```diff
@@ -62,12 +70,12 @@ def find_pipe(self, img):
         hsv = cv2.cvtColor(img, cv2.COLOR_BGR2HSV)
 
         channel = 2
-        norc = self.ncc(hsv[::2, ::2, channel], self.range[channel, 0])
+        #norc = self.ncc(hsv[::2, ::2, channel], self.range[channel, 0])
 
         color_mask = cv2.inRange(hsv, self.range[:, 0], self.range[:, 1])
-        ncc_mask = cv2.resize(np.uint8(norc > 0.9), None, fx=2, fy=2)
+        #ncc_mask = cv2.resize(np.uint8(norc >= 0), None, fx=2, fy=2)
 
-        mask = color_mask & ncc_mask
+        mask = color_mask  # & ncc_mask
```

Comment:
Yes, I forgot to tell you that. I tried to look into the problem but found that for temporary testing since I knew the exact orange, I could use just thresholding.
We should look into it